explainaboard/visualizers/performance_gap.py

```diff
@@ -20,24 +20,23 @@ def get_pairwise_performance_gap(
             orm_met.confidence_score_high = None
 
     fgr, fgr1, fgr2 = (unwrap(x.results.analyses) for x in (sys, sys1, sys2))
-    for fgr_all, fgr1_all, fgr2_all in zip(fgr, fgr1, fgr2):
-        for fgr_lev, fgr1_lev, fgr2_lev in zip(fgr_all, fgr1_all, fgr2_all):
-            if not isinstance(fgr_lev, BucketAnalysisResult):
-                continue
-            fgr_buks, fgr1_buks, fgr2_buks = (
-                cast(BucketAnalysisResult, x) for x in (fgr_lev, fgr1_lev, fgr2_lev)
-            )
-            for fgr_buk, fgr1_buk, fgr2_buk in zip(
-                fgr_buks.bucket_performances,
-                fgr1_buks.bucket_performances,
-                fgr2_buks.bucket_performances,
+    for fgr_lev, fgr1_lev, fgr2_lev in zip(fgr, fgr1, fgr2):
+        if not isinstance(fgr_lev, BucketAnalysisResult):
+            continue
+        fgr_buks, fgr1_buks, fgr2_buks = (
+            cast(BucketAnalysisResult, x) for x in (fgr_lev, fgr1_lev, fgr2_lev)
+        )
```

Comment:
This casting should not be necessary since `isinstance` in the previous statement already narrows the type.